### PR TITLE
Making some BE progress

### PR DESCRIPTION
**Pull request by facundoj**
- Adding user manager
- Adding retros manager
- Some sockets work
- Create retro functionality
- Log out

### CODE REVIEW

File: src/static/pages/retro/routes.js [hunk 1]
```diff
@@ -26,10 +28,24 @@ router.get('/:room', authorization.restrictToLoggedIn, function(req, res, next)
...
+        }
+
+        setupSocket(client, room);
+    });
+
+    function setupSocket(clientSocket, room) {
+        clientSocket.join(room);
+
+        clientSocket.emit('connection:accepted', room);
+        ns.breadcast.to(room);
```
Comment (gsas2) [inline src/static/pages/retro/routes.js]: ns.breadcast ?
Should it be clientSocket.broadcast.to(room).emit( someMessage)?

File: src/static/model/retros/manager.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// todo: When he have persistance, populate this pool
+var retros = {};
+
+function Retro(name, ownerUsername, participants) {
+    this.name = name;
+    this.owner = ownerUsername;
+    this.participants = participants || [];
+    this.isActive = true;
```
Comment (gsas2) [inline src/static/model/retros/manager.js]: Maybe we can have some kind of status, and use isActive as a method.
Cause will also need to know on what step of the retro we are (i.e.: adding items, voting, adding action items, closed)

**Comment by gsas2** [comment]
Approved by PEER ;)

File: src/static/model/retros/manager.js (hunk 2 above)
Comment (facundoj) [inline src/static/model/retros/manager.js]: I like it

File: src/static/model/retros/manager.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        }
+
+        var retro = new Retro(name, owner, participants);
+        retros[name] = retro;
+
+        return true;
+    },
+
+    /**
+     * Determines if a retro y active
```
Comment (gsas2) [inline src/static/model/retros/manager.js]: IS active :yum:

**Pull request merged**